packages/cli/src/main_parser_factory.test.ts

```diff
@@ -16,6 +17,11 @@ void describe('main parser', { concurrency: false }, () => {
     assert.match(output, /generate\s+Generates post deployment artifacts/);
   });
 
+  void it('shows version', async () => {
+    const output = await commandRunner.runCommand('--version');
+    assert.match(output, RegExp(version));
```

Comment:
Can this just be `assert.equal(output, version)`?

Reply:
Unfortunately, no. It'd have the following error

![image](https://github.com/aws-amplify/amplify-backend/assets/11983489/7ada1ee3-4898-44f4-bd92-7b9229f5e626)

Reply:
Done 33af02788